abi/vfabia64/vfabia64.rst

```diff
@@ -1489,6 +1489,12 @@ Masked signature generation for ``[not]inbranch`` clause.
 |``#pragma omp declare simd notinbranch`` |No     |Yes     |Yes   |No      |
 +-----------------------------------------+-------+--------+------+--------+
 
+Return value and masking
+------------------------
+
+The inactive lanes of return values of masked functions is ``undef``.
```

Comment:
`undef` sounds like LLVM terminology and isn't used elsewhere in the doc.  Maybe just "are undefined"?

Should we document this for arguments too?

Reply:
Done.